carma_wm_ctrl/src/WMBroadcasterNode.cpp

```diff
@@ -62,6 +66,8 @@ int WMBroadcasterNode::run()
   control_msg_pub_= cnh_.advertise<cav_msgs::TrafficControlRequest>("outgoing_geofence_request", 1, true);
   //Check Active Geofence Publisher
   active_pub_ = cnh_.advertise<cav_msgs::CheckActiveGeofence>("active_geofence", 200, true);
+  //publish TCM acknowledgement after processing TCM
+  tcm_ack_pub_ = cnh_.advertise<cav_msgs::MobilityOperation>("outgoing_geofence_ack", 1, true);
```

Comment:
Probably a good idea to set the queue size here greater than 1 maybe some multiple of the publication number like 2*msgnum so that messages published in the loop don't get dropped